### ISSUE TITLE
Matomo 4.6.0 reports 'vendor/lox/xhprof/bin/xhprofile' as problematic for File Integrity

### ISSUE THREAD

**Issue by mig5** [piwik/piwik#18418]
## Expected Behavior

After upgrading from Matomo 4.5.0 to 4.6.0 (swapping out the codebase with the new one from the tarball, and just retaining key files like config.ini.php), I expected to see no File Integrity issues in the Matomo admin interface.

## Current Behavior

The File Integrity area of the Matomo admin dashboard (System Check page) reported `vendor/lox/xhprof/bin/xhprofile` as being unexpected.

```
File integrity check failed and reported some errors. You should fix this issue and then refresh this page until it shows no error.
Files were found in your Matomo, but we didn't expect them.
--> Please delete these files to prevent errors. <--

File to delete: vendor/lox/xhprof/bin/xhprofile


To delete all these files at once, you can run this command:
rm "/var/www/html/piwik_4.6.0/vendor/lox/xhprof/scripts/xhprofile.php"
```

It recommended to remove it so I did. Note also the different file in the above command example (scripts/xhprofile.php) even though it's a different file marked as unexpected.

## Possible Solution

Either don't ship this file in the 4.6.0 tarball, or add it to list of expected files?

## Steps to Reproduce (for Bugs)

Install 4.6.0 from http://builds.matomo.org/matomo-4.6.0.tar.gz

## Your Environment

* Matomo Version: 4.6.0
* PHP Version: `PHP 7.2.34-28+ubuntu18.04.1+deb.sury.org+1 (cli) (built: Nov 19 2021 06:36:36) ( NTS )`
* Server Operating System: Ubuntu 18.04
* Browser: Firefox 78.15.0esr (64-bit)
* Operating System: Debian 10 (QubesOS)

**Comment by sgiehl** [comment]
@mig5 would you mind checking if that happens with 4.6.1 as well?

**Comment by prbt2016** [comment]
Hello,

I get the same issue as above while starting installation for Matomo 4.6.1 i.e : 

 File integrity check failed and reported some errors. You should fix this issue and then refresh this page until it shows no error.



```
Files were found in your Matomo, but we didn't expect them.
--> Please delete these files to prevent errors. <--

File to delete: vendor/lox/xhprof/bin/xhprofile

To delete all these files at once, you can run this command:
rm "/{{PATH}}/{{TO}}/{{MATOMO}}/vendor/lox/xhprof/bin/xhprofile"

```


![image](https://user-images.githubusercontent.com/17871448/144221145-24410056-cb7e-41b4-bbf2-738aee55f153.png)

Could you please check and fix?

**Comment by rogamoore** [comment]
I also have the same in 4.6.1

The file in `vendor/lox/xhprof/bin/xhprofile` seems to be a symbolic link. It points to `vendor/lox/xhprof/scripts/xhprofile.php`. For the latter, there is an entry in the manifest:

```php
		"vendor/lox/xhprof/scripts/xhprofile.php" => array("318", "aada44020058ead73cc477808fa9c4f6"),
```

So I guess, the manifest generation script should handle symbolic links differently.

**Comment by tsteur** [comment]
Some files are also being reported in https://github.com/matomo-org/matomo/issues/18415#issuecomment-984676727
![image](https://user-images.githubusercontent.com/273120/144485250-081b5d9f-ce64-47ce-8d57-fb3da03de956.png)
and https://github.com/matomo-org/matomo/issues/18411#issuecomment-984201606 
![image](https://user-images.githubusercontent.com/273120/144485308-1db73942-6c9a-4417-9940-8e79bcea8085.png)

I will create an issue to double check if there's a possibility to delete these files automatically. Maybe in the mean time an update script could try to remove the no longer needed files so not everyone sees these messages and gets confused?

**Comment by tsteur** [comment]
can this one be closed maybe?

**Comment by sgiehl** [comment]
I think so. The next release should at least solve a couple of the files in the lists above. All others should be fixed with 4.7.0 then.
